src/Microsoft.Sbom.Api/Executors/LicenseInformationService.cs

```diff
@@ -30,6 +30,11 @@ public LicenseInformationService(ILogger log, IRecorder recorder, HttpClient htt
     }
 
     public async Task<List<string>> FetchLicenseInformationFromAPI(List<string> listOfComponentsForApi)
+    {
+        return await FetchLicenseInformationFromAPI(listOfComponentsForApi, DefaultTimeoutInSeconds);
```

Comment:
The default is already implemented in the version without a timeout, so there's no need for `DefaultTimeoutInSeconds`
```suggestion
        return await FetchLicenseInformationFromAPI(listOfComponentsForApi);
```

Reply:
Hello Dave. I'm sorry, I don't follow on this one. If we remove `DefaultTimeoutInSeconds` from this method call as suggested, it just recursively calls itself until a Stack Overflow occurs. Did you perhaps mean modifying the call in `LicenseInformationFetcher.FetchLicenseInformationAsync(List<string> listOfComponentsForApi)` instead to call `LicenseInformationService.FetchLicenseInformationFromAPI(List<string> listOfComponentsForApi)` instead of `LicenseInformationFetcher.FetchLicenseInformationAsync(List<string> listOfComponentsForApi, int timeoutInSeconds)`?


```
// LicenseInformationFetcher.cs
public async Task<List<string>> FetchLicenseInformationAsync(List<string> listOfComponentsForApi)
{
    return await FetchLicenseInformationAsync(listOfComponentsForApi, DefaultTimeoutInSeconds);
}
```
to this?
```
// LicenseInformationFetcher.cs
public async Task<List<string>> FetchLicenseInformationAsync(List<string> listOfComponentsForApi)
{
    return await licenseInformationService.FetchLicenseInformationFromAPI(listOfComponentsForApi);
}
```